packages/office-ui-fabric-react/src/components/Persona/PersonaPresence/PersonaPresence.base.tsx

```diff
@@ -64,6 +62,7 @@ export class PersonaPresenceBase extends BaseComponent<IPersonaPresenceProps, {}
       <div
         className={ classNames.presence }
         style={ coinSizeWithPresenceStyle }
+        title={ presenceDescriptors![presence] }
```

Comment:
Hmm this doesn't seem like the right design. Are we sure we want a tooltip here?

@mtennoe for visibility

Reply:
Hm, yes, might not be the right design.
Do we want this mainly for screen-reader support, or is the actual accessibility design here to show a tooltip?
If we only want screen-reader support, we can set an aria-label instead

Reply:
My inspiration was the presence implementation in Teams.